src/aws/lambda-encode/src/encodeTimedTextTrack.js

```diff
@@ -40,10 +40,22 @@ module.exports = async (objectKey, sourceBucket, filename) => {
       // Transform the source key to the format expected for destination keys:
       // 630dfaaa-8b1c-4d2e-b708-c9a2d715cf59/timedtexttrack/dba1512e-d0b3-40cc-ae44-722fbe8cba6a/1542967735_fr
       // 👆 becomes 👇
-      // 630dfaaa-8b1c-4d2e-b708-c9a2d715cf59/timedtext/1542967735_fr
+      // 630dfaaa-8b1c-4d2e-b708-c9a2d715cf59/timedtext/1542967735_fr.vtt
       Key: `${objectKey.replace(/\/timedtexttrack\/.*\//, '/timedtext/')}.vtt`,
     })
     .promise();
+
+  await s3.
+    copyObject({
+      Bucket: destinationBucket,
+      Key: `${objectKey.replace(/\/timedtexttrack\/.*\//, '/timedtext/')}`,
```

Comment:
i feel weird that the name of the original file is exactly the same as the transformed file except only the extension... put them in a `source` or `source_files` directory or something like that?

Reply:
:+1: